lib/models/cartoonCats.js

```diff
@@ -0,0 +1,27 @@
+const pool = require('../utils/pool');
+
+module.exports = class Cat {
+  id;
+  name;
+  year;
+  lives;
+  isSidekick;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.year = row.year;
+    this.lives = row.lives;
+    this.isSidekick = row.isSidekick;
```

Comment:
This row comes back as undefined because SQL normalizes column names to lowercase. So, to get the data that you want here, you have to rename `row.isSidekick` to `row.issidekick`

That being said, convention is that multi-word column names in SQL be written in snake case. So, to do that, you'd have to rename that part to `row.is_sidekick` here, and in your `setup.sql` file. But leave the `this.isSidekick` part as is.